### DISCUSSION TITLE
quickdraw seems too easy to obtain

### DISCUSSION THREAD

**Discussion by alivarastepour** [community/community#60922] [Other Feature Feedback, Questions, & Ideas]
I accidentally closed my own pull request some time after I opened it, and now I have a quickdraw badge. what's that about?

**Comment by enginestein** [comment]
Quickdraw badge is obtained when you open a pull request and close it within 5 minutes of opening.

**Comment by enginestein** [comment]
I think so.

**Comment by NomanGhiasy** [comment]
It's a badge you get after when you create a new PR and they are lost within 5 minutes of opening.

**Reply by enginestein** [reply]
Not lost, but if we close them.

**Comment by YasinDehfuli** [comment]
If you need help to get Quickdraw achievement or any other achievements step by step by screenshots:

you can use this repository to learn:

ENG : https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.quickdraw.md

Persian : https://github.com/4xmen/Get-Github-Achievements/blob/main/README.fa.md
